ha/const.py

```diff
@@ -53,7 +53,7 @@
 CLUSTER_CONFSTORE_NODES_KEY="nodes"
 
 # Cortx commands
-CORTX_CLUSTER_NODE_ADD="cortx cluster add node --nodeid <node> --username <user> --password <secret>"
+CORTX_CLUSTER_NODE_ADD="cortx cluster add node --nodeid=<node> --username=<user> --password=<secret>"
```

Comment:
@ajay-paratmandali Will this need the help string update?

Reply:
Updated